csp-mixin/signals/gcpvpc.libsonnet

```diff
@@ -0,0 +1,295 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: this.filteringSelector,
+    groupLabels: ['job'],
+    instanceLabels: ['project_id'],
+    aggLevel: 'group',
+    discoveryMetric: {
+      stackdriver: 'stackdriver_networking_googleapis_com_location_networking_googleapis_com_fixed_standard_tier_usage',
+    },
+    signals: {
+      //
+      // Fixed Usage Tier
+      //
+      // available labels: ['bandwidth_policy_id', 'location', 'project_id', 'traffic_source']
+      gcpvpc_fixed_usage_tier_network_egress: {
+        name: 'Fixed standard tier network egress',
+        description: 'Rate in bytes of network egress on the fixed standard tier.',
+        type: 'gauge',
+        unit: 'bps',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_networking_googleapis_com_location_networking_googleapis_com_fixed_standard_tier_usage{%(queriesSelector)s}',
+            aggKeepLabels: ['traffic_source'],
+          },
+        },
+      },
+
+      //
+      // Overview counters
+      //
+      gcpvpc_services_in_use_count: {
+        name: 'Total services in use',
+        description: 'Count of unique service names (including "UNKNOWN") with ingress or egress traffic.',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          stackdriver: {
+            expr: 'count(count by (service_name) (stackdriver_google_service_gce_client_networking_googleapis_com_google_service_response_bytes_count{%(queriesSelector)s} OR stackdriver_google_service_gce_client_networking_googleapis_com_google_service_request_bytes_count{%(queriesSelector)s}))',
```

Comment:
Why do you use `count by service_name` OR the `same metric value `itself? What is the purpose?